### PR TITLE
add email authentication part 2

### PR DESCRIPTION
**Pull request by kian1213**
Hello sir, please check my pull request.This is the continuation of the email authentication.

Thanks po

### CODE REVIEW

**Comment by codegracie** [comment]
Okay

File: config/application.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+development:
+  smtp:
+    host: smtp.mandrillapp.com
+    port: 587
+    user_name: christian.angel.g.galamay@framgia.com
```
Comment (dieunb) [inline config/application.yml]: you should store it on environment variable linux system

File: config/application.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    password: vekxYcb3ROT7Uffwk0KVlQ
```
Comment (dieunb) [inline config/application.yml]: same above

File: config/environments/development.rb [hunk 3]
```diff
@@ -22,6 +22,15 @@
   # Raise an error on page load if there are pending migrations.
   config.active_record.migration_error = :page_load
 
+  config.action_mailer.smtp_settings = {
+    :address   => APP_CONFIG[:smtp][:host],
```
Comment (dieunb) [inline config/environments/development.rb]: Convention, should as `address: APP_CONFIG[:smtp][:host],`

File: config/environments/development.rb [hunk 4]
```diff
@@ -22,6 +22,15 @@
...
+    :port      => APP_CONFIG[:smtp][:port],
```
Comment (dieunb) [inline config/environments/development.rb]: same above

File: config/environments/development.rb [hunk 5]
```diff
@@ -22,6 +22,15 @@
...
+    :user_name => APP_CONFIG[:smtp][:user_name],
```
Comment (dieunb) [inline config/environments/development.rb]: same above

File: config/environments/development.rb [hunk 6]
```diff
@@ -22,6 +22,15 @@
...
+    :password  => APP_CONFIG[:smtp][:password],
```
Comment (dieunb) [inline config/environments/development.rb]: same above

File: config/environments/development.rb [hunk 7]
```diff
@@ -22,6 +22,15 @@
...
+    :authentication => :plain,
```
Comment (dieunb) [inline config/environments/development.rb]: same above

File: config/environments/development.rb [hunk 8]
```diff
@@ -22,6 +22,15 @@
...
+    :enable_starttls_auto => true
```
Comment (dieunb) [inline config/environments/development.rb]: same above

**Pull request merged**